src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorTests.cs

```diff
@@ -39,6 +39,40 @@ public async Task MapAction_NoParam_StringReturn(string source, string expectedB
         Assert.Equal(expectedBody, body);
     }
 
+    [Fact]
+    public async Task MapGet_NoParam_StringReturn_WithFilter()
```

Comment:
Not necessarily for this PR, but what do you think about having a small set of tests (1-5?) that baseline the generated code. That way it is easy in PRs to see what changes are being made to the generated code. In my opinion, it helps spot bugs if I can read the generated code.

Reply:
I went ahead and added the infrastructure to do this and applied it to this test.